### PR TITLE
DEV-3388-new-go-exercise-displayp-subject

### PR DESCRIPTION
**Pull request by jotapero**
- docs(displayP): created new subject
- PR did not exist

### CODE REVIEW

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/displays/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## displayp
+
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/displays/README.md]: fixed

File: subjects/displays/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+### Instructions
+
+Write a program that displays an `P` character on the standard followed by newlines (`'\n'`)
```
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/displays/README.md]: fixed

File: subjects/displays/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jotapero) [reply subjects/displays/README.md]: fixed

**Review by tiagocollot — APPROVED** [review]
LGTM!

File: subjects/displayp/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+$ go run . | cat -e
+P$
+$
+$ go run . "a" "b" | cat -e
+P$
+$
+$ go run . "a" "b" "c" | cat -e
+P$
+$
+```
```
Comment (eslopfer) [inline subjects/displayp/README.md:19]: Please run prettier

**Review by eslopfer — DISMISSED** [review]

**Pull request merged**